config/crd/patches/webhook_in_appwrappers.yaml

```diff
@@ -0,0 +1,16 @@
+# The following patch enables a conversion webhook for the CRD
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: appwrappers.workload.codeflare.dev
+spec:
+  conversion:
```

Comment:
dito: Do we really have a conversion webhook in place?

Reply:
removed.